### ISSUE TITLE
Now Playing info wraps overflow instead of scrolling it

### ISSUE THREAD

**Issue by ohitsdylan** [bang-olufsen/create#126]
I've installed the latest version of HifiBerryOS on a RaspberryPi 3B+. The music information on the "now playing" area does not scroll like it did in previous releases, but rather, just wraps onto a new line.

I have the Pi running the HifiBerryOS instance and am using Firefox to access the UI over the network (Ethernet). I am using the Dev Tools to set my screen resolution to 800x480 to mimic the size of the 7" RaspberryPi touchscreen. Doing this did not trouble the text scrolling feature in the past.

Just to be safe, I checked the repo for the existence of the text scrolling function and it is indeed there.
I have attached a GIF of the text wrapping, if it is at all useful.

Thank you!

![Text wrapping example](https://user-images.githubusercontent.com/55002846/102536720-017aa480-4078-11eb-8a82-f4e9eb03162a.gif)

**Comment by tuomashamalainen** [comment]
Hi Dylan,

the Now Playing screen was redesigned during the summer and wrapping is expected behaviour. In wide layouts there is more space for text next to the album cover, so we utilise this space for multiline text instead of scrolling it. In compact layouts, where the text is below the album cover, text will scroll just as before.

There might be instances where text might be so long that it overflows the space we have, and these are not yet properly addressed.

Best,
Tuomas from Bang & Olufsen

**Comment by ohitsdylan** [comment]
Wow, thank you for the quick response!

I just made my browser window smaller and sure enough it started scrolling. I will add this scrolling to the larger layout in a personal fork, then.

Thanks again!

**Comment by tuomashamalainen** [comment]
No problem at all! This is also a matter of taste obviously, so I can see why you would want the scrolling text back. Let me know if you have any questions.

Best,
Tuomas from Bang & Olufsen

**Comment by ohitsdylan** [comment]
While I was playing around with the text scrolling scripts, I also tried adjusting the size of the album artwork and the width of the track metadata container and came to understand the change from scrolling to text wrapping. It definitely fits the overall feel of the rest of the application, so I decided to keep it.

I felt the text container was a bit small and wrapped a little too often, so I gave it some more room and made the artwork smaller.

![Image of UI alteration](https://user-images.githubusercontent.com/55002846/103015997-594d5b80-450f-11eb-8ceb-8a74f8e2b607.png)
